libuuu/gen_ver.sh

```diff
@@ -7,7 +7,7 @@ if [ "${APPVEYOR_BUILD_VERSION}" = "" ];
 then
 	echo build not in appveyor
 else
-	git tag uuu_${APPVEYOR_BUILD_VERSION}
+	git tag -m"uuu ${APPVEYOR_BUILD_VERSION}" uuu_${APPVEYOR_BUILD_VERSION}
```

Comment:
This code only run at AppVeryor build machine to keep build version info is consistent.

I just choose stable version and tag in github release page.  Look like Github just create simple tag on release page.